scripts/regenerate_docs

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+docker pull pseudomuto/protoc-gen-doc
```

Comment:
Obviously I would prefer that this was part of bazel but I won't block on that!

Reply:
IIRC We need to update docker rules because they're still using Python 2.7, I'll look into updating this after that is done!